scripts/cybershake/install_cybershake_fault.py

```diff
@@ -229,31 +223,25 @@ def install_fault(
             ignore_vm_qpqs_files=ignore_vm_qpqs_files,
         )
 
+        if (
+            root_params_dict is None
+            or fault_params_dict is None
+            or sim_params_dict is None
+            or vm_params_dict is None
+        ):
+            # Something has gone wrong, returning without saving anything
+            logger.critical(f"Critical Error some params dictionary are None")
+            return
```

Comment:
I was about to say it might be nice to print something if it's an error like this. But tbh that was the original code behaviour and I've never seen this error before. So all good!

Reply:
This is only going to raise if the vm_params nt doesn't match with the nt calculated from the sim duration and the gmsim version root_params dt value. I'm not sure this part of the PR needs to go in?
It doesn't seem to change the functionality (Other than the logging statement)

Reply:
this kind of error only happened in ancient times when we are not using proper config reading. what i did was simply moved it a few line above, and removed the lines that sets them all to None, if another check failed.
The feature @jpa198  referring to is line 236~243, now it simply returns instead of setting dictionary to None (this was usefully when we were still using imports instead of load_yaml)

Reply:
so these line now serve as checking if the dictionaries are None, instead of a code exit.

Reply:
If this only happened when we were importing dicts instead of loading yamls, then why is it still here?
I was actually referring to line 142(old)/112(new) of install_shared. 236 relates to comparing flo, which now has its own return